Frontend/Shared/NavMenu.razor

```diff
@@ -26,9 +26,18 @@
                 </NavLink>
             </div>
             <div class="nav-item">
-                <NavLink href="profile" ActiveClass="link-active">
-                    <ion-icon name="person-circle-outline"></ion-icon>
-                </NavLink>
+                <div class="dropdown" role="button" data-bs-toggle="dropdown">
```

Comment:
This does not work for me, please check again if it does for you. Probably put a NavLink as each dropdown item. The href should point to the actual destination, not just profile.

Reply:
With that the styling (blue icon) will probably not work out of the box so remember to put that back in.